app/src/main/java/com/sopt/now/compose/presentation/SignUp/SignUpViewModel.kt

```diff
@@ -0,0 +1,51 @@
+package com.sopt.now.compose.presentation.SignUp
+
+
+import android.content.ContentValues.TAG
+import android.util.Log
+import androidx.compose.runtime.mutableStateOf
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.sopt.now.compose.domain.AuthRepository
+import com.sopt.now.compose.domain.model.AuthEntity
+import kotlinx.coroutines.launch
+import retrofit2.HttpException
+
+class SignUpViewModel(
+    private val authRepository: AuthRepository,
+) : ViewModel() {
+    val signUpResult: MutableLiveData<Boolean> = MutableLiveData()
+    val errorMessage: MutableLiveData<String> = MutableLiveData()
+    val userId: MutableLiveData<String> = MutableLiveData()
+
+    var signUpId = mutableStateOf("")
+    var signUpPw = mutableStateOf("")
+    var signUpName = mutableStateOf("")
+    var signUpPhone = mutableStateOf("")
```

Comment:
```suggestion
    var id by rememberSaveable { mutableStateOf("") }
    var password by rememberSaveable { mutableStateOf("") }
    var nickName by rememberSaveable { mutableStateOf("") }
    var mbti by rememberSaveable { mutableStateOf("") }
```

by remember키워드를 통해 객체를 생성후 생된 객체를 기억하고 재사용 할 수 있습니다!
컴포즈에서의 상태관리 패턴이에여